include/matx/transforms/cub.h

```diff
@@ -76,6 +76,7 @@ typedef enum {
   CUB_OP_SELECT_IDX,
   CUB_OP_UNIQUE,
   CUB_OP_ARG_REDUCE,
+  CUB_OP_DUAL_ARG_REDUCE,
```

Comment:
Quad?

Reply:
Changed to CUB_OP_SINGLE_ARG_REDUCE and CUB_OP_DUAL_ARG_REDUCE